tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>The printed message is misleading here, not sure we should have it all.</p><p> </p><p>In any case, perhaps you should use <code>set_config()</code>  here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Do you still need the <code>reset_config</code>  here?</p>

Reply:
<p>No, will change that</p>